sys/net/gnrc/conn/gnrc_conn.c

```diff
@@ -38,7 +75,7 @@ int gnrc_conn_recvfrom(conn_t *conn, void *data, size_t max_len, void *addr, siz
                 }
                 LL_SEARCH_SCALAR(pkt, l3hdr, type, conn->l3_type);
                 if (l3hdr == NULL) {
-                    msg_send_to_self(&msg); /* requeue invalid messages */
+                    msg_send_to_self(&msg); /* requeue invalid messages, wouldn't this make the thread just read the message 3 times and return? */
```

Comment:
I don't get the comment. There can be other `msg_receive()`s in the thread. Also: line length ;-)